packs/equipment/axe-of-the-dwarven-lords.json

```diff
@@ -95,11 +95,11 @@
                 "selector": "strike-damage"
```

Comment:
```suggestion
                "selector": "{item|id}-damage"
```

Reply:
Whoops! Good catch - I didn't notice it.